aether-kernel/aether/kernel/api/migrations/0104_schemadecorator_topic_name.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.2.1 on 2019-05-15 09:57
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations
+
+def migrate__update_schemadecorator(apps, schema_editor):
+    SchemaDecorator = apps.get_model('kernel', 'SchemaDecorator')
+    for sd in SchemaDecorator.objects.all():
+        sd.save() # default topic_name to { name: [schemadecorator.name] }
```

Comment:
flake8 warning... at least 2 spaces between the piece of code and the inline comment ;-)